services/scraper/search_keyword.go

```diff
@@ -0,0 +1,86 @@
+package scraper
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/gocolly/colly/v2"
+)
+
+type SearchKeywordService struct {
+	Keyword string
+
+	synchronous  bool
```

Comment:
For boolean, prefer using the prefix `is` or `has` to distinguish these from other types of attributes:

```suggestion
	isSynchronous  bool
```